docs/consume-packages/Package-References-in-Project-Files.md

```diff
@@ -165,7 +165,44 @@ Conditions can also be applied at the `ItemGroup` level and will apply to all ch
 </ItemGroup>
 ```
 
+## GeneratePathProperty
+
+*Version **4.9** or above*
```

Comment:
As we didn't insert into 15.9, mentioned in this issue: https://github.com/NuGet/Home/issues/8374#issuecomment-515607739 , we didn't support it for 15.x.
Do we need to change it to `*This feature is available with NuGet **4.9** or above and with Visual Studio 2019 **16.0** or above.*` ?

Reply:
Nice catch.